packages/application-extension/src/index.tsx

```diff
@@ -372,9 +401,11 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
           return mode === 'single-document' || mode === 'multiple-document';
         },
         execute: args => {
+          let ariaLiveRegion = document.getElementById('commands-aria-live');
           const mode = args['mode'] as string;
           if (mode === 'single-document' || mode === 'multiple-document') {
             labShell.mode = mode;
+            ariaLiveRegion!.append(trans.__(`${mode} mode`));
```

Comment:
We cannot use JS templated strings with translations, it needs to use `%` and pass the variable in the second argument, see: https://jupyterlab.readthedocs.io/en/4.0.x/extension/internationalization.html

Reply:
Changes made.